### DISCUSSION TITLE
https://espixelstickwebflasher.from-ct.com:5000/ is Down?

### DISCUSSION THREAD

**Discussion by dutch-anykey** [forkineye/ESPixelStick#811] [General]
Hi,

Is the Webflash site down or do I have a local problem here.

br
Maurice

**Comment by MartinMueller2003** [comment]
We lost power last night. It seems someone was not careful and took out a pole. Once power was restored,  the Internet service provider realized they have a problem and have been working to fix it since. At the moment, the site is up and running but the messages from the ISP indicate we may have interruptions throughout the next 12 hours as they continue fixing the damage.
